docs/source/concept_models/vivarium_swissre_colorectalcancer/concept_model.rst

```diff
@@ -338,15 +338,15 @@ Positive for FOBT does not necessarily mean you have colorectal cancer
      - Medium-risk
      - Fecal occult blood test (FOBT)
      - every year
-     - 33%-75%
-     - 100%
+     - 68% (95%CI 57%-79%)
```

Comment:
Can you convert this into a distribution, mean, standard deviation, and addl parameters, if relevant?  E.g. Normal with mean 68, and stdev XXX, or Truncated Log Normal with log_mean XX, stdev YY, lb ZZ, ub WW, or whatever you think is best.

Reply:
Will do!